templates/apollo-rest-starter/src/config/config.ts

```diff
@@ -1,5 +1,5 @@
 import knex from 'knex'
-import * as config from '../../config.json'
+import * as config from '../../graphback.json'
```

Comment:
We can create issue about this problem to see if we can decompose config from the database.